packages/binutils.rb

```diff
@@ -4,31 +4,19 @@ class Binutils < Package
   description 'The GNU Binutils are a collection of binary tools.'
   homepage 'http://www.gnu.org/software/binutils/'
   version '2.30'
-  source_url 'https://ftpmirror.gnu.org/binutils/binutils-2.30.tar.xz'
+  source_url 'https://ftp.gnu.org/gnu/binutils/binutils-2.30.tar.xz'
```

Comment:
We want the original url since it optimizes the mirror to download based on locale.